contracts/ics/base/Cargo.toml

```diff
@@ -9,7 +9,7 @@ repository = "https://github.com/synapseweb3/ibc-ckb-contracts"
 [dependencies]
 ckb-std = "0.13.0"
 ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "58f256c" }
-axon-tools-riscv = { version = "0.1.1", features = ["proof"] }
+axon-tools-riscv = { git = "https://github.com/axonweb3/axon.git", package = "axon-tools", rev = "06340ba4", version = "0.1.1", features = ["proof"] }
```

Comment:
We don't have a separate `axon-tools-riscv` now. How about just use `axon-tools = {...}`.